DfE.FindInformationAcademiesTrusts/Pages/Privacy.cshtml.cs

```diff
@@ -0,0 +1,7 @@
+using DfE.FindInformationAcademiesTrusts.Pages.Shared;
+using Microsoft.AspNetCore.Authorization;
+
+namespace DfE.FindInformationAcademiesTrusts.Pages;
+
+[AllowAnonymous]
+public class PrivacyModel : ContentPageModel;
```

Comment:
This isn't something that matters hugely, but maybe instead of creating empty models for the privacy and Accessibility pages could we create an Anonymous Page model those could use instead?